var/spack/repos/builtin.mock/packages/splice-b/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class SpliceB(Package):
+    """Simple package with one optional dependency"""
```

Comment:
Are we dealing with splicing in any way? If not can we rename this and avoid touching tests unrelated to this fix?